src/internal/m365/collection/exchange/export.go

```diff
@@ -51,6 +52,8 @@ func streamItems(
 			id := item.ID()
 			name := id + ".eml"
 
+			itemCtx := clues.Add(ictx, "item_file_name", id)
```

Comment:
nit: "file name" is an interpretation of how the value will be used, not actually what it is.  that may drift over time/future implementation.
```suggestion
			itemCtx := clues.Add(ictx, "stream_item_id", id)
```